packages/analytics/src/controllers/users.js

```diff
@@ -1,6 +1,20 @@
-import { userList } from '../models/users.js'
+import { userList, userCount } from '../models/users.js'
 
-// eslint-disable-next-line import/prefer-default-export
 export async function userListController(req, res) {
-  res.render('users', { users: await userList() })
+  const pageNumber = req.query.page || 1
+  const itemsPerPage = 50
+
+  try {
+    const totalUsers = await userCount()
+    const maxPages = Math.ceil(totalUsers / itemsPerPage)
```

Comment:
I've seen this operations for three times. Consider make it a function to generate the page list